contracts/exchange/SmartWalletSwapImplementation.sol

```diff
@@ -29,27 +27,6 @@ contract SmartWalletSwapImplementation is SmartWalletSwapStorage, ISmartWalletSw
         }
     }
 
-    function updateKyberProxy(IKyberProxy _proxy) external onlyAdmin {
```

Comment:
So removing the updateKyberProxy and updateUniswapRouters, it means that we will need to redeploy the SmartWallet contracts if they have an update as well.